wordpress-mobile/MediaEditor-iOS
# Add Filters

### To test

1. Run the Example app
2. Tap the filters icon
3. Play with some filters and apply them
4. The image should be updated with the filter

**Important**: running in the simulator can be slow or cause some weird image outputs. Better to try in a real device.

PR submission checklist:

- [x] I have considered adding unit tests where possible — I'm really not sure if it worths to test the `CIImage` implementation. Open to feedback. :)
